scripts/extract-hvd-content.ts

```diff
@@ -3,19 +3,25 @@ import path from 'path'
 import { fetchGithubArchiveZip } from 'lib/fetch-github-archive-zip'
 import { isDeployPreview } from 'lib/env-checks'
 
+export const BASE_REPO_CONFIG = {
+	owner: 'hashicorp',
+	ref: process.env.CURRENT_GIT_BRANCH || 'main',
+	repo: 'hvd-docs',
+	contentPath: '/content',
```

Comment:
I think moving all content in the `hvd-docs` repo into a `content` dir is the right move in the long term, especially when we want to add local data support since we can just point to a directory where content lives instead of looking at the entire repo.